ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,813 @@
+              }
+            },
+            "location": {
+              "description": "Holdings record effective location",
+              "type": "object",
+              "properties": {
+                "permanentLocation": {
+                  "type": "object",
+                  "description": "Permanent shelving location in which an item resides",
+                  "properties": {
+                    "name": {
+                      "description": "Name of the (shelf) location",
+                      "type": "string"
+                    },
+                    "campusName": {
+                      "description": "The name of the campus, the second-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    },
+                    "libraryName": {
+                      "description": "The name of the library, the third-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    },
+                    "institutionName": {
+                      "description": "The name of the institution, the first-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "name",
+                    "campusName",
+                    "libraryName",
+                    "institutionName"
+                  ]
+                },
+                "temporaryLocation": {
+                  "type": "object",
+                  "description": "Temporary location, shelving location, or holding which is a physical place where items are stored, or an Online location",
+                  "properties": {
+                    "name": {
+                      "description": "Name of the (shelf) location",
+                      "type": "string"
+                    },
+                    "campusName": {
+                      "description": "The name of the campus, the second-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    },
+                    "libraryName": {
+                      "description": "The name of the library, the third-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    },
+                    "institutionName": {
+                      "description": "The name of the institution, the first-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "name",
+                    "campusName",
+                    "libraryName",
+                    "institutionName"
+                  ]
+                }
+              },
+              "required": [
+                "permanentLocation",
+                "temporaryLocation"
```

Comment:
How can temporary location be required when it is not required for holdings in general?

Reply:
I checked and fixed reirement for payload's fields.